classes/markdown.lua

```diff
@@ -54,10 +54,7 @@ SILE.registerCommand("bulletlist", function (_, content)
 end)
 
 SILE.registerCommand("link", function (_, content)
-  -- SILE.settings.temporarily(function ()
```

Comment:
Did this form stop working when we changed `\verbetim:font` or did you change it preemptively?

Reply:
See my comment from a few minutes ago, I basically re-enabled it as it looked like it should be working now. Didn't test yet, though.

Reply:
Oh right. I see now.